creator-node/src/config.js

```diff
@@ -280,6 +280,12 @@ const config = convict({
     env: 'delegatePrivateKey',
     default: null
   },
+  isRegisteredOnURSM: {
```

Comment:
do we need to set this? i don't think we'd ever want to have anyone ever pass this in as an env var

Reply:
good call, fixed